src/pages/Community/Components/PostDisplayList.tsx

```diff
@@ -52,9 +54,11 @@ export default function PostDisplayList({
           {/* Large screen post button */}
           <div className="hidden sm:block">
             <Link to={`/c/${communityId}?post=new-post`}>
-              <button type="button" className="btn-filled rounded-3xl py-2.5">
+              <button
+                type="button"
+                className="button-action px-5 rounded-3xl py-2.5">
                 <div className="flex items-center gap-1 -ml-1 ">
-                  <PlusIcon className="h-5" /> Post
+                  <PlusIcon className="h-5 -ml-1" /> Post
```

Comment:
Instead of using negative margin, remove the items-center from the parent flex box and maybe align things at the beginning.

Maybe in this case negative margin is the only way to go, but I've learned that in 99% of cases negative margins are not a good solution.